src/integration-test/tests/citizen/claimantResponse/full_admission_test.ts

```diff
@@ -15,122 +15,127 @@ const claimantResponseSteps: ClaimantResponseSteps = new ClaimantResponseSteps()
 const checkAndSendPage: ClaimantCheckAndSendPage = new ClaimantCheckAndSendPage()
 const confirmationPage: ClaimantConfirmation = new ClaimantConfirmation()
 
-Feature('Claimant Response').retry(3)
+if (process.env.FEATURE_ADMISSIONS !== undefined && JSON.parse(process.env.FEATURE_ADMISSIONS)) {
```

Comment:
@timja this line is of interest too... enabling if toggled

Reply:
JSON.parse? isn't it just a string?

Reply:
you would think but I had to do this to work locally (docker-compose) and on pipeline...maybe not - i will revisit this before any merge